src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    public Machine() {
+    }
+
+    public abstract void doWork();
+
+    public abstract void stopWork();

```

Comment:
```suggestion
    void doWork();

    void stopWork();

```
They are public and abstract by default